### PR TITLE
TM-4082: Update/admin remarks insertion removal

### PR DESCRIPTION
**Pull request by pveland621**
ACs:
1. Removing an insertion from the `Remark Insertions` list below should now remove that specific insertion from the `Remark Description` input box

Only note is me using the `eslint-disable-next-line no-plusplus` - I couldn't get the code to work any other way besides using the increment operator

### CODE REVIEW

**Comment by M00staff** [comment]
OK - yea, this should work for this specific bit of functionality + the demo 

but don't close out the ticket until the submit button passes a payload to Redux, which calls the future endpoint (dunno if one already exists) ...there were also some console errors that needed to be cleaned up, and some unnecessary lodash usage - and maybe adding a success/error toast ... you should be able to find most of that in my PR (https://github.com/MetaPhase-Consulting/State-TalentMAP/pull/2552/files)

THEN we can close this ticket, and make a separate ticket and work on the specific backend part

File: src/Components/AdministratorPage/EditRemark/EditRemark.jsx [hunk 1]
```diff
@@ -26,6 +26,12 @@ const EditRemark = (props) => {
     const returnArray = [...rmrkInsertionList];
     returnArray.splice(i, 1);
     setRmrkInsertionList(returnArray);
+
+    const regex = new RegExp('{[^}]*}', 'g');
```
Comment (elizabeth-jimenez) [inline src/Components/AdministratorPage/EditRemark/EditRemark.jsx:30]: 😍

File: src/Components/AdministratorPage/EditRemark/EditRemark.jsx [hunk 2]
```diff
@@ -26,6 +26,12 @@ const EditRemark = (props) => {
...
+    let x = 0;
+    // eslint-disable-next-line no-plusplus
+    const result = descriptionInput.replace(regex, (match) => (++x === i + 1 ? '' : match));
```
Comment (elizabeth-jimenez) [inline src/Components/AdministratorPage/EditRemark/EditRemark.jsx:33]: We can do something like this instead
    const returnArray = [...rmrkInsertionList];
    const removeMe = returnArray[i];
    returnArray.splice(i, 1);
    setRmrkInsertionList(returnArray);
    const result = descriptionInput.replace(removeMe, '');
    setDescriptionInput(result);

Comment (elizabeth-jimenez) [reply src/Components/AdministratorPage/EditRemark/EditRemark.jsx:33]: no we can't - Patrick enlightened me!

**Review by elizabeth-jimenez — APPROVED** [review]

**Comment by pveland621** [comment]
so just so we're all on the same page - final verdict is this: 

1. merge and deploy this little fix for the demo tomorrow
2. create new branch/PR to finish out the rest of the FE work using most of Moose's code
3. start working on 4121 which is the BE portions (on a new branch/PR)

File: src/Components/AdministratorPage/EditRemark/EditRemark.jsx (hunk 2 above)
Comment (elizabeth-jimenez) [inline src/Components/AdministratorPage/EditRemark/EditRemark.jsx:33]: One change we can make is `(x++ === i ? '' : match)` you were having to add 1 to i bc u were checking against x after the increment

**Pull request merged**